Cargo.toml

```diff
@@ -265,6 +265,7 @@ crossbeam-channel = "0.5.0"
 [[example]]
 name = "hello_world"
 path = "examples/hello_world.rs"
+doc-scrape-examples = true
```

Comment:
why is this needed? it's not mentioned in the page you linked

Reply:
Yeah, so it builds locally without this, but IIRC docs.rs needs it to be aware of which examples to read from

Reply:
I did this because cargo said it will need this when the examples use dev dependencies:


```console
$ cargo +nightly doc -Zunstable-options -Zrustdoc-scrape-examples
warning: Rustdoc did not scrape the following examples because they require dev-dependencies: hello_world, bloom_2d, move_sprite, rotation, mesh2d, mesh2d_manual, mesh2d_vertex_color_texture, 2d_shapes, custom_gltf_vertex_attribute, 2d_gizmos, sprite, sprite_flipping, sprite_sheet, text2d, texture_atlas, transparency_2d, pixel_perfect, 3d_scene, 3d_shapes, generate_custom_mesh, anti_aliasing, 3d_gizmos, atmospheric_fog, fog, blend_modes, lighting, lines, ssao, spotlight, bloom_3d, load_gltf, tonemapping, orthographic, parenting, pbr, parallax_mapping, render_to_texture, shadow_biases, shadow_caster_receiver, skybox, spherical_area_lights, split_screen, texture, transparency_3d, two_passes, update_gltf_scene, vertex_colors, wireframe, no_prepass, animated_fox, morph_targets, animated_transform, cubic_curve, custom_skinned_mesh, gltf_skinned_mesh, custom_loop, drag_and_drop, empty, empty_defaults, headless, logs, plugin, plugin_group, return_after_run, thread_pool_resources, no_renderer, without_winit, asset_loading, custom_asset, custom_asset_io, hot_asset_reloading, async_compute, external_source_external_thread, audio, audio_control, decodable, spatial_audio_2d, spatial_audio_3d, pitch, log_diagnostics, custom_diagnostic, ecs_guide, apply_deferred, component_change_detection, custom_query_param, event, fixed_timestep, generic_system, hierarchy, iter_combinations, parallel_query, removal_detection, run_conditions, startup_system, state, system_piping, system_closure, system_param, timers, alien_cake_addict, breakout, contributors, game_menu, char_input_events, gamepad_input, gamepad_input_events, gamepad_rumble, keyboard_input, keyboard_modifiers, keyboard_input_events, mouse_input, mouse_input_events, mouse_grab, touch_input, touch_input_events, text_input, reflection, generic_reflection, reflection_types, trait_reflection, scene, custom_vertex_attribute, post_processing, shader_defs, shader_material, shader_prepass, shader_material_screenspace_texture, shader_material_glsl, shader_instancing, animate_shader, compute_shader_game_of_life, array_texture, texture_binding_array, bevymark, many_animated_sprites, many_buttons, many_cubes, many_gizmos, many_foxes, many_glyphs, many_lights, many_sprites, transform_hierarchy, text_pipeline, scene_viewer, gamepad_viewer, nondeterministic_system_order, 3d_rotation, scale, transform, translation, borders, button, display_and_visibility, window_fallthrough, font_atlas_debug, overflow, overflow_debug, relative_cursor_position, size_constraints, text, text_debug, flex_layout, text_wrap_debug, grid, transparency_ui, z_index, ui, ui_scaling, ui_texture_atlas, viewport_debug, clear_color, low_power, multiple_windows, scale_factor_override, screenshot, transparent_window, window_settings, resizing, minimising, window_resizing, fallback_image
    If you want Rustdoc to scrape these examples, then add `doc-scrape-examples = true`
    to the [[example]] target configuration of at least one example.
```